### PR TITLE
py-libensemble: add v1.4.0, v1.4.1

### PR DESCRIPTION
**Pull request by shuds13**

### CODE REVIEW

**Review by jlnav — APPROVED** [review]

**Review by jlnav — APPROVED** [review]

**Comment by shuds13** [comment]
@tldahlgren I'm not sure why this is blocked as I have an approval? Should I be able to pull this in.

**Comment by alecbcs** [comment]
@shuds13 this isn't blocked, just hasn't yet received a review from someone on the core team.

**Review by alecbcs — APPROVED** [review]
Verified Checksums and Python dependencies. Looks good to me. Thanks @shuds13!

**Pull request merged**
